src/main/java/com/courtcanva/ccfranchise/config/SecurityConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.courtcanva.ccfranchise.config;
+
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.cors.CorsConfiguration;
+
+import java.util.List;
+
+@Slf4j
+@Setter
+@Configuration
+@EnableWebSecurity
```

Comment:
I guess you might have thoughts that maybe we don't need this annotation as well. If you did then you are right. I tried to run the application without it and everything just works fine(I am also shocked by the result). These "Enable" series annotations were popular before the Spring boot was born. Now they are not at all since Spring Boot encourages "Convention over Configuration", which means a lot of functions are enabled by default.

Reply:
I just found out @EnableWebSecurity is necessary to configure HTTP in the latest version of Spring Security. So never mind, I will resolve the conversation.